app/scripts/views/search.js

```diff
@@ -10,7 +10,9 @@ var Col     = require('react-bootstrap/Col');
 
 var Search  = React.createClass({
     render: function(){
-        var zipcode = location.href.split(/search\//)[1].match(/\d+/g)[0];
+        var query   = location.href.split(/search\//)[1].split('/');
```

Comment:
I guess there is another way to retrieve the query from the Router component itself. Can you check their documentations?